ci/kind/test-e2e-kind.sh

```diff
@@ -107,6 +125,11 @@ case $key in
 esac
 done
 
+if [[ $cleanup_only == "true" ]];then
+  $TESTBED_CMD destroy kind
+  exit 0
```

Comment:
should `trap "quit" INT EXIT` be moved after it, otherwise it would try to clean up twice?

Reply:
the returned code is `0`, it won't be called twice in old codes, but `quit` is refined now, so I moved the trap after this. thanks.